COMPILE-PROTOS.sh

```diff
@@ -17,11 +17,6 @@
 
 go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
 
-protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative openapiv2/*.proto
-protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative openapiv3/*.proto
-protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative discovery/*.proto
 protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative plugins/*.proto
-protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative extensions/*.proto
 protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative surface/*.proto
-protoc -I . -I ./third_party --go_out=. --go_opt=paths=source_relative metrics/*.proto
```

Comment:
Removed the directories which have proto definitions in gnostic-models, and noticed `metrics` was compiling twice.